apisix/plugin.lua

```diff
@@ -433,6 +438,8 @@ local function meta_filter(ctx, plugin_name, plugin_conf)
                          " plugin_name: ", plugin_name)
         return true
     end
+
+    ctx[match_cache_key] = ok
```

Comment:
I think we can't cache the result of the filter. Since we have a different phase for every plugin like "rewrite", "access" e.g.

Reply:
`filter` works better at the plugin level and is executed at each phase, which will increase the difficulty for users to use.